graph/vertex_cover.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+  @author [Rajat Garg](https://github.com/Rajat379)
+  @file
+
+  Implementation of a vertex cover in a graph.
+*/
+
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <list>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```